docs/startingcommands.rst

```diff
@@ -157,7 +159,9 @@ ls command
 
 We use *ls* command to *list* the files and directories inside any given
 directory. If you use *ls* command without any argument, then it will work on
-the current directory. We will see few examples of the command below.::
+the current directory. We will see few examples of the command below.
+
```

Comment:
Both are valid RST :)

Reply:
Indeed, but we'll get different outputs. If a code-directive (::) present just after period, in output we'll get
a colon (:) just after period, like this:
```
the current directory. We will see few examples of the command below.:
```
Whereas, with later case, we'll get the output like this:
```
the current directory. We will see few examples of the command below.
```